docs/containers-storage.conf.5.md

```diff
@@ -56,10 +56,9 @@ $ restorecon -R -v /NEWSTORAGEPATH
 A common use case for this field is to provide a local storage directory when user home directories are NFS-mounted (podman does not support container storage over NFS).
 
 **imagestore**=""
-  image storage path (the default is assumed to be the same as `graphroot`).
-  Path of imagestore different from `graphroot`, by default storage library stores all images in `graphroot` but if `imagestore` is provided it will store newly pulled images in provided `imagestore` but will keep using `graphroot` for everything else. If user is using `overlay` driver then images which were already part of `graphroot` will still be accessible ( Internally storage library will mount `graphroot` as an `additionalImageStore` to allow this behaviour ).
+  image storage path (the default is assumed to be the same as `graphroot`). Path of imagestore different from `graphroot`. By default images in the storage library are stored in the `graphroot`. If `imagestore` is provided, newly pulled images will be stored in the `imagestore` location. All other storage continues to be stored in the `graphroot`. When using `overlay` driver images previously stored in the `graphroot` continue to be accessible ( Internally storage library mounts `graphroot` as an `additionalImageStore` to allow this behaviour ).
```

Comment:
I still find this hard to grok. It's missing a couple of commas, and is a little confusing, but it's better than it was, so thank you.